app/models/group.rb

```diff
@@ -8,7 +7,11 @@ class Group < ActiveRecord::Base
   belongs_to :neighborhood
 
   has_many :user_groups
-  has_many :users, through: :user_groups
+  has_many :users, through: :user_groups do
```

Comment:
Using this scope, we're able to do `current_group.users.leaders` to query all the leaders of that group.